stable/traefik/values.yaml

```diff
@@ -1,9 +1,24 @@
 # Default values for Traefik
 image: traefik
 imageTag: 1.3.8
-serviceType: LoadBalancer
-loadBalancerIP:
-#loadBalancerSourceRanges: []
```

Comment:
We cannot remove and replace the above. To do so is to introduce breaking changes.

We can be intelligent about how we structure _new_ keys, but existing ones must remain.